tests/Functional/Driver/OCI8/ConnectionTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional\Driver\OCI8;
+
+use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Exception\DriverException;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Tests\TestUtil;
+use PHPUnit\Framework\Attributes\RequiresPhpExtension;
+use Throwable;
+
+#[RequiresPhpExtension('oci8')]
```

Comment:
We probably want to test it with `pdo_oci` as well. Instead of requiring a PHP extension, please use `TestUtils::isDriverOneOf()`. Otherwise, if the extension is available at runtime, this test will run with any other drivers and fail.

Reply:
Done